tracer/src/Datadog.Trace/AppSec/Transports/Http/HttpTransport.Framework.cs

```diff
@@ -24,38 +25,34 @@ internal class HttpTransport : ITransport
 
         public Func<string, string> GetHeader => key => context.Request.Headers[key];
 
-        public void AddRequestScope(Guid guid)
+        public IContext GetAdditiveContext() => context.Items[WafKey] as IContext;
```

Comment:
Dumb question but will this key be always present in the context?

Reply:
No, we rely on it returning null when the WafKey isn't present, this isn't a new time with this PR, it just been moved from lower down the file.